src/AssetsPage/AssetsTable/index.jsx

```diff
@@ -38,6 +79,41 @@ class AssetsTable extends React.Component {
     }
   }
 
+  onSortClick(columnKey) {
+    // const prevColumnKey = this.state.sortState.columnKey;
+    const columns = [...this.state.tableColumns];
+    const column = this.state.tableColumns.find(col => (col.key === columnKey));
+    let newDirection;
+
+    if (this.state.sortState.columnKey === columnKey) {
+      newDirection = this.state.sortState.direction === 'descending' ? 'ascending' : 'descending';
+      column.sortDirection = newDirection;
+
+      // then we want to make the appropriate API call
+    } else {
+      // is this the best way to do a line break?
```

Comment:
Re: how to do line breaks the best way in React

Reply:
Do you mean in code, or inside of display elements? In code, as long as we're somewhat consistent and readable I don't have a strong opinion. In display elements, I think `span` elements are correct, according to a quick google search.